src/components/contact/populateContact.js

```diff
@@ -0,0 +1,35 @@
+import img1 from '../../images/contact/img2.jpg';
+
+const populateContact = () => {
+  const contact = document.createElement('div');
+  contact.setAttribute('class', 'd-flex flex-column flex-wrap text-white');
+
+  const mapImage = document.createElement('img');
+  mapImage.setAttribute('class', 'map-image');
+  mapImage.src = img1;
+
+  const restaurantName = document.createElement('h2');
+  restaurantName.textContent = 'Kebab House Istanbul Restaurant';
+
+  const openHours = document.createElement('p');
+  openHours.textContent = 'Open hours: 08:00 - 22:00';
+
+  const phoneNumber = document.createElement('p');
+  phoneNumber.textContent = 'Phone number: +90 432 123 45 67';
+
+  const adress = document.createElement('p');
+  adress.textContent = 'Address: Buroko Street, Ataturk Avenue, Istanbul TURKEY';
+
+  const email = document.createElement('p');
+  email.textContent = 'Email: info@kebabhouseistanbul.com';
+
+  contact.appendChild(restaurantName);
```

Comment:
- [ ] **[OPTIONAL]** We can increase the speed and performance of appending children to our nodes by avoiding reflows (drawing the page every time you append a child to a certain div) using [`createDocumentFragment`](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment) function. 
Think of it as a temporary element created in memory that you append children instead of the target element, and once you are done with appending any number of children, you append this fragment to the target without affecting the performance (one reflow vs multiple reflows).

Let's check other modules to use the same technique.

Reply:
Thank you for your feedback, I will check it.